app/views/courses/_basic_details_tab.html.erb

```diff
@@ -286,13 +278,9 @@
         <p class="govuk-body">At the moment you can change:</p>
         <ul class="govuk-list govuk-list--bullet">
           <li><abbr class="app-text-decoration-underline-dotted" title="Special educational needs and disability">SEND</abbr> specialism</li>
-          <li>age range</li>
-          <li>outcome</li>
-          <li>full time or part time</li>
-          <li>course locations</li>
-          <li>application open date</li>
-          <li>course start date</li>
-          <li>GCSE requirements</li>
+          <% course.listing_basic_details.each do |details|%>
+              <li><%= "#{details}" %></li>
```

Comment:
Not a blocker, but I don't thinks details needed to be wrapped in double quotes
```
<%= details %>
```

Reply:
Thanks Al, i've changed it